test/byContract/IexecPoco/IexecPoco1.test.ts

```diff
@@ -206,9 +215,122 @@ describe('IexecPoco1', () => {
             });
         });
     });
-    // TODO
-    describe('Verify presignature', () => {});
-    describe('Verify presignature or signature', () => {});
+
+    describe('verifyPresignature', () => {
+        it('Should verify the correct presignature', async () => {
+            // Assume appProvider presigned the appOrderHash
+            await iexecPoco
+                .connect(appProvider)
+                .manageAppOrder(createOrderOperation(orders.app, OrderOperationEnum.SIGN));
+
+            expect(await iexecPocoContract.verifyPresignature(appProvider.address, appOrderHash)).to
```

Comment:
Would you be able to try something like `iexecPocoContract[verifySignatureFunction]` with `['verifyPresignature', 'verifyPresignatureOrSignature']` so you can remove next `describe('Verify presignature or signature', ..)` block ?